cli/src/pcluster/aws/ec2.py

```diff
@@ -123,6 +123,16 @@ def get_instance_ids_by_ami_id(self, image_id):
             for instance in result.get("Instances")
         ]
 
+    @AWSExceptionHandler.handle_client_exception
+    def get_image_shared_account_ids(self, image_id):
+        """Get account ids that image is shared with."""
+        return [
+            user.get("UserId")
+            for user in self._client.describe_image_attribute(Attribute="launchPermission", ImageId=image_id).get(
```

Comment:
Image can be shared to group as well, and this need to be checked, i.e.
```
    "LaunchPermissions": [
        {
            "Group": "Group",
            "UserId": "UserId"
        }
    ],
```
For example when an Image is public when `Group: All`

the return should be something 
```
[
            permission.get("UserId") or permission.get("Group") 
            for permission in self._client.describe_image_attribute(Attribute="launchPermission", ImageId=image_id).get("LaunchPermissions")
        ]
```

Reply:
done